.github/workflows/npm-publish.yml

```diff
@@ -4,20 +4,12 @@
 name: Publish package
 
 on:
-  workflow_run:
-    workflows: ['Tests']
-    branches: [main]
-    types:
-      - completed
-  push:
-    branches:
-      - '!*'
-    tags:
-      - 'v*'
+  release:
+    types: [created]
 
 jobs:
   publish-npm:
-    if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
+    if: contains(github.ref_name,'beta') == false
```

Comment:
The file `.github/workflows/npm-publish-canary.yml` could be entirely removed by making this one compatible with the two types of releases.

To do that, this condition should be removed and we need to define the package tag (`steps.uses.with.tag`) with the following condition: `${{ contains(github.ref_name,'beta') && 'beta' || 'latest' }}`

Reply:
It is working!

Reply:
🥳